.github/workflows/acceptance-tests.yml

```diff
@@ -12,7 +12,53 @@ on:
   pull_request: # you can run a specic job in your PR using GitHub labels
     types: [ labeled ]
 
+env:
+    FORCE_TEST_ACC: ${{ github.event_name == 'workflow_dispatch' || github.event_name == 'schedule' || github.event.label.name == 'run-testacc' || inputs.parent-event-name == 'release' }}
+    TF_LOG: ${{ vars.LOG_LEVEL }}
+    SKIP_TEST_EXTERNAL_CREDENTIALS: ${{ vars.SKIP_TEST_EXTERNAL_CREDENTIALS }}
+    ACCTEST_TIMEOUT: ${{ vars.ACCTEST_TIMEOUT }}
+    PARALLEL_GO_TEST: "20"
 jobs:  
+  search_index:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
Could we move `actions/checkout@v4` to `github/actions/acceptance-test-action/action.yml`??

Reply:
no, that was my first try but it can't fine the action file